CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 1.9.1-RC1
+
+## Enhancements
+
+- References librdkafka.redist 1.9.1 which includes an Apple M1 librdkafka build.
+- Enhanced the Avro Specific Deserializer to ignore the type namespace ([sergemat](https://github.com/sergemat)).
+
+
+## Fixes
+
+- The AdminClient poll loop no longer terminates when a request results in an error ([emasab](https://github.com/emasab)).
```

Comment:
No problem if you don't reference me, as we are part of the team. You could put an hashtag with the PR for the team members, and for the rest the username too.

Reply:
yeah, either way. when in doubt, give credit :-)